Dockerfile

```diff
@@ -9,5 +9,6 @@ RUN useradd -ms /bin/bash -d /vim-go vim-go
 USER vim-go
 WORKDIR /vim-go
 COPY . /vim-go/
+RUN make install
 
 ENTRYPOINT ["make"]
```

Comment:
We need to change this to `make test lint` probably as `make` also includes install

Reply:
The default target, `all`, does include `install`, but there's no harm in that and acts a good safety valve in case you do something like `docker run --rm -v my-empty-dir:/tmp vim-go-test`.

Reply:
The `install-vim` script doesn't do anything if this version is already installed:
https://github.com/fatih/vim-go/blob/master/scripts/install-vim#L48

Reply:
the default target has `install` unfortunately: https://github.com/fatih/vim-go/blob/master/Makefile#L1 Maybe we should remove it? would be better imho, anyone is free to install via `make install` later

Reply:
@Carpetsmoker fair enough :)

Reply:
I think `install` definitely belongs in `all`; I want to be able to `make` locally and have it do all the right things.

Reply:
It does seem to re-install Vim on the second run btw. I tested with both this PR and #1550 applied (probably better to do that in the same PR).

Reply:
I'm done it several times and do not see vim getting installed when the container is run.